README.md

```diff
@@ -971,6 +971,37 @@ Show native app rating dialog
 showAppRating = () => Promise<void>
 ```
 
+### increaseAppRatingTrigger
+
+<kbd>Available in B2P App version >=25.6</kbd>
+
+Request native app to increase appRating trigger value for a specific key
```

Comment:
I think a better explanation is needed in this doc. In a month I won't remember what's an app ratting trigger value key

Reply:
You are right. Added some more info here and in reset method.